admin/future_grants/V1.6.5__internamespace_object_ownership_prod.sql

```diff
@@ -0,0 +1,17 @@
+-- Grant future ownership of object types which potentially need
+-- internamespace privileges to proxy admin database role.
+-- SYNAPSE
+GRANT OWNERSHIP
+	ON FUTURE DYNAMIC TABLES
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.ALL_ADMIN;
+GRANT OWNERSHIP
+	ON FUTURE TASKS
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE
```

Comment:
Same thing here, I don't see tasks in this namespace. Are we anticipating having tasks here?

Reply:
I'll remove it.